Frontend/src/pages/dashboard/components/Farmer/FarmerDashCards.jsx

```
@@ -58,89 +58,103 @@ function FarmerDashCards() {
+    // Helper function to filter orders by date range
+    const filterOrdersByDate = (orders, startDate, endDate = null) => {
+      return orders.filter((order) => {
+        const orderDate = new Date(order.date);
+        return orderDate >= startDate && (!endDate || orderDate < endDate);
+      });
+    };
+
+    // Helper function to calculate total revenue from orders
+    const calculateTotalRevenue = (orders) => {
+      return orders.reduce((sum, order) => {
+        const price = parseFloat(order.price || 0);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)